fgd/point/filter/filter_momentum_timer_progress.fgd

```diff
@@ -0,0 +1,8 @@
+@FilterClass base(filter_base)
+	appliesto(MOMENTUM)
+= filter_momentum_timer_progress : "Filter a player based on their current track, stage, and checkpoint."
```

Comment:
Subtle wording change to account for the fact that non-player entities are subjected to this filter as well (but it still checks the player's timer state rather than some state owned by the entity itself) and also to match the wording of other filters:
```suggestion
= filter_momentum_timer_progress : "A filter that filters by the player's current track, stage, and checkpoint."
```
Since other filters are named things like filter_*activator*_name to reflect they filter based on the activator's properties, this could technically be named something like `filter_momentum_player_timer_progress` but that's probably overkill 😅